res/values/strings_pl.arb

```diff
@@ -418,36 +418,36 @@
   "change_wallet_alert_content" : "Czy chcesz zmienić obecny portfel na ${wallet_name}?",
 
   "creating_new_wallet" : "Tworzenie nowego portfela",
-  "creating_new_wallet_error" : "Pomyłka: ${description}",
+  "creating_new_wallet_error" : "Błąd: ${description}",
 
   "seed_alert_title" : "Uwaga",
-  "seed_alert_content" : "Ziarno to jedyny sposób na odzyskanie portfela. Zapisałeś to?",
+  "seed_alert_content" : "Fraza Seed to jedyny sposób na odzyskanie portfela. Zapisałeś ją?",
   "seed_alert_back" : "Wróć",
   "seed_alert_yes" : "Tak",
 
   "exchange_sync_alert_content" : "Poczekaj, aż portfel zostanie zsynchronizowany",
 
   "pre_seed_title" : "WAŻNY",
-  "pre_seed_description" : "Na następnej stronie zobaczysz serię ${words} słów. To jest Twoje unikalne i prywatne ziarno i jest to JEDYNY sposób na odzyskanie portfela w przypadku utraty lub awarii. Twoim obowiązkiem jest zapisanie go i przechowywanie w bezpiecznym miejscu poza aplikacją Cake Wallet.",
-  "pre_seed_button_text" : "Rozumiem. Pokaż mi moje nasienie",
+  "pre_seed_description" : "Na następnej stronie zobaczysz serię ${words} słów. To jest Twoje unikalna i prywatna fraza seed i jest to JEDYNY sposób na odzyskanie portfela w przypadku utraty lub awarii telefonu. Twoim obowiązkiem jest zapisanie go i przechowywanie w bezpiecznym miejscu (np. na kartce w SEJFIE).",
```

Comment:
Slightly different wording here; should be ok

Reply:
I thought it would be better for non-technical users. It is "hold your seed phrase e.g. on paper in vault" instead of "out of cake wallet app" which can be interpreted as to store it in e.g. notes app